include/circt/Dialect/Verif/VerifOps.td

```diff
@@ -267,14 +267,17 @@ def YieldOp : VerifOp<"yield", [
 }
 
 //===----------------------------------------------------------------------===//
-// Verification intent ops
+// Verification Intent Ops
 //===----------------------------------------------------------------------===//
 
 def FormalOp : VerifOp<"formal", [
-  NoTerminator, RegionKindInterface, HasParent<"mlir::ModuleOp">, 
-  IsolatedFromAbove, Symbol
+  IsolatedFromAbove,
+  NoTerminator,
+  RegionKindInterface,
+  SingleBlock,
+  Symbol,
 ]> {
-  let summary = "defines a formal verification test";
+  let summary = "A formal test case";
   let description = [{
```

Comment:
Adding a tool shouldn't be changing these, consider breaking out and committing separately?

Reply:
Good point :grin: